scripts/zones/Arrapago_Reef/TextIDs.lua

```diff
@@ -1,23 +1,29 @@
 -- Variable TextID   Description text
 
 -- General Texts
-ITEM_CANNOT_BE_OBTAINED = 6381; -- You cannot obtain the item <item> come back again after sorting your inventory
-          ITEM_OBTAINED = 6387; -- Obtained: <item>
-           GIL_OBTAINED = 6388; -- Obtained <number> gil
-       KEYITEM_OBTAINED = 6390; -- Obtained key item: <keyitem>
- FISHING_MESSAGE_OFFSET = 7048; -- You can't fish here
+ITEM_CANNOT_BE_OBTAINED =  6381; -- You cannot obtain the item <item> come back again after sorting your inventory
+          ITEM_OBTAINED =  6387; -- Obtained: <item>
+           GIL_OBTAINED =  6388; -- Obtained <number> gil
+       KEYITEM_OBTAINED =  6390; -- Obtained key item: <keyitem>
+ FISHING_MESSAGE_OFFSET =  7048; -- You can't fish here
 
 -- Assault
-           CANNOT_ENTER = 8444; -- You cannot enter at this time.  Please wait a while before trying again.
-              AREA_FULL = 8445; -- This area is fully occupied. You were unable to enter.
-         MEMBER_NO_REQS = 8449; -- Not all of your party members meet the requirements for this objective.  Unable to enter area.
-         MEMBER_TOO_FAR = 8453; -- One or more party members are too far away from the entrance.  Unable to enter area.
+           CANNOT_ENTER =  8444; -- You cannot enter at this time.  Please wait a while before trying again.
+              AREA_FULL =  8445; -- This area is fully occupied. You were unable to enter.
+         MEMBER_NO_REQS =  8449; -- Not all of your party members meet the requirements for this objective.  Unable to enter area.
+         MEMBER_TOO_FAR =  8453; -- One or more party members are too far away from the entrance.  Unable to enter area.
 
 -- Other Texts
-            YOU_NO_REQS = 7894; -- You do not meet the requirements to enter the battlefield
-        NOTHING_HAPPENS =  119; -- Nothing happens...
-               RESPONSE = 7328; -- There is no response...
+            YOU_NO_REQS =  7894; -- You do not meet the requirements to enter the battlefield
+        NOTHING_HAPPENS =   119; -- Nothing happens...
+               RESPONSE =  7328; -- There is no response...
+         DOOR_IS_LOCKED = 40225; -- The door is locked. You might be able to open it with a <item>.
+        DOOR_IS_LOCKED2 = 40226; -- The door is locked. You might be able to open it with a <item> or a <item>.
+             KEY_BREAKS = 40227; -- The <item> breaks.
+           LOCK_SUCCESS = 40231; -- <player> successfully opened the lock with the <item>!
+              LOCK_FAIL = 40232; -- <player> failed to open the lock with the <item>...
```

Comment:
7464

Reply:
I didn't use POLUtils, I got numbers from retail capture

Reply:
Retail will use a separate id for each npc where in dsp we can away with reusing them most of the time

Reply:
Subtract 0x8000 (32768) from the MessageID you get from retail 0x2A packets for the the values used by DSP/found by POLUtils.

Reply:
I shoulda have known when seeing them past the signed int16 limit. previously only whitegate has used multiple string tables.